### PR TITLE
Added "Spacebucks" barsign

### PR DESCRIPTION
**Pull request by ProPandaBear**
Added the animated barsign for "Spacebucks" because what do our corporate overlords love more than franchises? You can't escape them, even in space!

**Media**
Screenshot 2023-04-22 120049

* [x]  I have added screenshots/videos to this PR showcasing its changes ingame, or this PR does not require an ingame showcase

**Changelog**

🆑

add: The "Spacebucks" barsign was added.

### CODE REVIEW

**Comment by DogZeroX** [comment]
Can you remove "4 the emprah" while at it, feels like it'll be sussy copyright.

File: Resources/Textures/Structures/Wallmounts/barsign.rsi/meta.json [hunk 1]
```diff
@@ -461,21 +461,40 @@
...
-          0.2,
-          0.2,
-          0.2,
-          0.2,
-          0.2,
-          0.2,
-          0.2,
-          0.2,
-          0.2
+        "name": "enginechange",
```
Comment (deltanedas) [inline Resources/Textures/Structures/Wallmounts/barsign.rsi/meta.json:464]: indentation got trolled i think

Comment (ProPandaBear) [reply Resources/Textures/Structures/Wallmounts/barsign.rsi/meta.json:464]: Good catch, fixing that now.

Comment (ProPandaBear) [reply Resources/Textures/Structures/Wallmounts/barsign.rsi/meta.json]: Should now be resolved.

**Review by keronshb — CHANGES_REQUESTED** [review]
Does Spacebucks come from any ss13 server or is it a custom sprite? If it's not in either of the copyright commits then please add another line of copyright. Otherwise this is fine.

**Comment by ProPandaBear** [comment]
> Does Spacebucks come from any ss13 server or is it a custom sprite? If it's not in either of the copyright commits then please add another line of copyright. Otherwise this is fine.

It's a totally custom sprite I made. I can totally add in a line of copyright, to be honest I just wasn't sure how to attribute copyright to myself in this context.

File: Resources/Textures/Structures/Wallmounts/barsign.rsi/meta.json [hunk 2]
```diff
@@ -7,6 +7,7 @@
   "license": "CC-BY-SA-3.0",
   "copyright": "https://github.com/vgstation-coders/vgstation13/commit/1fbb5e417d0200fe60c6b628656a371093d2d715",
   "copyright": "https://github.com/tgstation/tgstation/commit/68aade356cf53032f206cb5a887581ac7d166cae",
+  "copyright": "https://github.com/space-wizards/space-station-14/commit/b301d3ac26cb2301048eda4535667c28e2283d41",
```
Comment (keronshb) [inline Resources/Textures/Structures/Wallmounts/barsign.rsi/meta.json:10]:
```suggestion
  "copyright": "https://github.com/space-wizards/space-station-14/commit/b301d3ac26cb2301048eda4535667c28e2283d41 - Made by ProPandaBear for SS14",
```

You don't need to add your own commit for this but I guess that works. You should add something like:

Made by githubname for SS14

Comment (ProPandaBear) [reply Resources/Textures/Structures/Wallmounts/barsign.rsi/meta.json]: Awesome, thanks for the help. Went ahead and committed your suggestion.

**Comment by keronshb** [comment]
One more change, the CL shouldn't have Added: To the game

It should mention what it adds in the description:

- add: The "Spacebucks" barsign was added.

Or something similar to that.

**Comment by ProPandaBear** [comment]
> One more change, the CL shouldn't have Added: To the game
> 
> It should mention what it adds in the description:
> 
> * add: The "Spacebucks" barsign was added.
> 
> Or something similar to that. This is mostly due to how the CL bot works.

Just updated it!

**Review by keronshb — APPROVED** [review]

**Pull request merged**